docs/user-guide/helm.md

```diff
@@ -1,56 +1,76 @@
 
-## Prerequisites
+## Before You Begin
 
-The Ambassador Edge Stack Helm chart is hosted by Datawire and published at `https://www.getambassador.io`. Start by adding this repo to your helm client with:
+The Ambassador Edge Stack Helm chart is hosted by Datawire and published at `https://www.getambassador.io`.
+
+Start by adding this repo to your helm client with the following command:
 
 ```bash
 helm repo add datawire https://www.getambassador.io
 ```
 
-## First Time Installation
+## Install with Helm
 
-If you are installing the Ambassador Edge Stack for the first time on your host, complete the following directions:
+When you run the Helm chart, it installs the Ambassador Edge Stack. You can deploy it with either the Helm 2 or Helm 3 chart.
 
-1. Create the `ambassador` namespace for the Ambassador Edge Stack:
+1. If you are installing the Ambassador Edge Stack **for the first time on your cluster**, create the `ambassador` namespace for the Ambassador Edge Stack:
 
-    ```
-    kubectl create namespace ambassador
-    ```
+   ```
+   kubectl create namespace ambassador
+   ```
 
-2. If you are using Helm 3, install the Ambassador Edge Stack Chart with the following command:
+2. **Helm 3 users:** Install the Ambassador Edge Stack Chart with the following command:
 
-    ```
-    helm install ambassador --namespace ambassador datawire/ambassador
-    ```
+   ```
+   helm install ambassador --namespace ambassador datawire/ambassador
+   ```
 
-If you are using Helm 2, use the following command instead:
+3. **Helm 2 users**: Install the Ambassador Edge Stack Chart with the following command:
 
+   ```
+   helm install --name ambassador --namespace ambassador datawire/ambassador
+   ```
+
+4. Finish the installation by running the following command: `edgectl install`
+5. Provide an email address when prompted to receive notices if your domain or TLS certificate is about to expire.
+
+Your terminal should print something similar to the following:
+
+    ```shell
+    $ edgectl install
+    -> Installing the Ambassador Edge Stack 1.0.
+    -> Existing Ambassador Edge Stack installation detected.
+    -> Automatically configuring TLS.
+    Please enter an email address. We’ll use this email address to notify you prior to domain and certification expiration [None]: john@example.com.
+    -> Obtaining a TLS certificate from Let’s Encrypt.
+
+    Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit https://random-word.edgestack.me to access your Edge Stack installation and for additional configuration.
     ```
-    helm install --name ambassador --namespace ambassador datawire/ambassador
-    ```
+
+The command `edgectl` is part of [Edge Control](/reference/edge-control), which automatically configures TLS for your instance and provisions a domain name for your Ambassador Edge Stack.
```

Comment:
Did you mean `edgectl install` is part of Edge Control? Even that sounds a bit odd to me. Edge Control is `edgectl`; they are different spellings for the same thing. Maybe something like this can get across the important point:

```markdown
The [Edge Control](/reference/edge-control) command 
you ran (`edgectl install`) provisions a domain name 
for your Ambassador Edge Stack instance and 
automatically configures TLS.
```